tests/Functional/Query/QueryBuilderTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Query;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Platforms\DB2Platform;
+use Doctrine\DBAL\Platforms\MariaDb1027Platform;
+use Doctrine\DBAL\Platforms\MySQL57Platform;
+use Doctrine\DBAL\Platforms\SqlitePlatform;
+use Doctrine\DBAL\Query\QueryBuilder;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+use Doctrine\DBAL\Types\Types;
+
+use function get_class;
+use function sprintf;
+
+final class QueryBuilderTest extends FunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        $platform = $this->connection->getDatabasePlatform();
+        if (! $this->supportsPlatform($platform)) {
+            self::markTestSkipped(sprintf('Skipping since connected to %s', get_class($platform)));
```

Comment:
Please always include a reason _why_ we skip the test.

Reply:
improved the skip message to 'Skipping, because platform %s does not support locks'